src/app/home/tabs/feed/feed.component.html

```diff
@@ -1,21 +1,27 @@
 <ion-content class="ion-padding">
   <ion-refresher (ionRefresh)="handleRefresh($event)" slot="fixed">
-    <ion-refresher-content>
-    </ion-refresher-content>
+    <ion-refresher-content> </ion-refresher-content>
   </ion-refresher>
   <div class="center-container">
-    @for (item of feedState.locationRiddles(); track item.locationRiddleId) {
+    <div>
+      @if(!feedState.loading()){ @for (item of feedState.locationRiddles();
+      track item.locationRiddleId) {
       <app-location-riddle-post
         [locationRiddleImage]="item.locationRiddleImage"
         [username]="getUsername(item.userId)"
         [createdAt]="item.createdAt"
         [rating]="item.rating"
         [comments]="item.comments"
       ></app-location-riddle-post>
-    } @empty {
+      } @empty {
       <div class="base-container">
         <p>No location riddles to solve...</p>
       </div>
-    }
+      } } @else {
```

Comment:
I guess this works, however also have the loading spinner from ionic-refresher-content, can we remove one can we use loading to complete the refresher one? If loading is true and we swipe down we will see two spinners

Reply:
the problem is that on init the "no riddles" message is displayed since the initial state is an empty list. It is also not possible to programmatically trigger the refresh spinner. Therefore, I added an if block that removes the refresh block on initial loading.